app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
```

Comment:
Replace these three `from ... import` with one and use `()` according to the [checklist](https://github.com/Artemh59/py-lockdown-cafe/blob/develop/checklist.md#code-style).